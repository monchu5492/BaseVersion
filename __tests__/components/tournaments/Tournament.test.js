import toJSON from 'enzyme-to-json';
import { shallow, mount } from 'enzyme';
import { compose } from 'redux';
import Tournament from '../../../src/components/ui/Tournament';


describe('<Tournament /> UI Component', () => {
  const shallowExpect = compose(expect, toJSON, shallow);

  it('Renders correct properties', () =>
    shallowExpect(<Tournament
      id={1}
      name="First Tourney"
    />)
      .toMatchSnapshot());
});
