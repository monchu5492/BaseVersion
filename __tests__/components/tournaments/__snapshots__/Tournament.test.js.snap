// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tournament /> UI Component Renders correct properties 1`] = `
<section
  className="tournament"
>
  <h1
    onClick={[Function]}
  >
    First Tourney
  </h1>
  <button
    onClick={[Function]}
  >
    <FaTrashO />
  </button>
  <div
    className="tournament"
    onClick={[Function]}
  />
</section>
`;
